packages/astro/src/core/preview/index.ts

```diff
@@ -33,9 +45,29 @@ export default async function preview(config: AstroConfig, { logging }: PreviewO
 			return;
 		}
 
-		send(req, req.url!.substr(base.length - 1), {
+		const requrl = removeBase(base, req.url!);
+
+		// Redirect /docs to /docs/
+		if(requrl === '/' && !req.url!.endsWith('/')) {
```

Comment:
we have `devOptions.trailingSlash: "always" | "never" | "ignore";` as a config value, I'd imagine that this would read from that behavior and act accordingly?

Reply:
What is the behavior of that supposed to be? Does "always" mean a trailing slash is required and 404 or does it mean that we forward to the trailing slash (this behavior)? 

Having trouble finding docs or where this was implemented.

Reply:
This is helpful: https://github.com/withastro/astro/pull/1197/files